<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tools::FspPrinterDialogAmounts</class>
 <widget class="QWidget" name="Tools::FspPrinterDialogAmounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Activit√©</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>C/V...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Autres1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Autres2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Utils::ModernDateEditor" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IK/MD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Nb</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Depassement</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="8">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::ModernDateEditor</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/widgets/moderndateeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
